src/Runner.Listener/CommandSettings.cs

```diff
@@ -249,6 +250,24 @@ public string GetStartupType()
             return GetArg(Constants.Runner.CommandLine.Args.StartupType);
         }
 
+        public ISet<string> GetLabels(string description)
```

Comment:
do we need to take the `description` as input? feels like we have the full context of generating description here.

Reply:
To display this -> `This runner will have the following labels: 'self-hosted', 'Windows', 'X64'`, which has dynamic labels info.

Reply:
it only use `{VarUtil.OS}', '{VarUtil.OSArchitecture}` which are static property?